src/main/java/com/github/jmchilton/blend4j/galaxy/HistoriesClient.java

```diff
@@ -37,6 +38,20 @@
   HistoryExport exportHistory(String historyId);
   
   /**
+   * Deletes the given History from Galaxy (this will not purge).
+   * @param historyId  The id of the History to delete.
+   * @return A {@link ClientResponse} for this request.
+   */
+  ClientResponse deleteHistoryResponse(String historyId);
```

Comment:
My question still stands: as an API user, how do I know which one I should be using? Is it necessary to have `HistoryDeleteResponse` if all of that information is embedded in `ClientResponse`?